pkg/vpcmodel/grouping.go

```diff
@@ -275,30 +294,47 @@ func (g *GroupConnLines) groupExternalAddressesForDiff(thisMinusOther bool) erro
 	var res []*groupedConnLine
 	var connRemovedChanged connectivityDiff
 	if thisMinusOther {
-		connRemovedChanged = g.d.cfg1ConnRemovedFrom2
+		connRemovedChanged = g.diff.cfg1ConnRemovedFrom2
 	} else {
-		connRemovedChanged = g.d.cfg2ConnRemovedFrom1
+		connRemovedChanged = g.diff.cfg2ConnRemovedFrom1
 	}
 	for src, endpointConnDiff := range connRemovedChanged {
 		for dst, connDiff := range endpointConnDiff {
 			connDiffString := connDiffEncode(src, dst, connDiff)
-			connsEmpty := connDiff.conn1.IsEmpty() && connDiff.conn2.IsEmpty()
-			err := g.addLineToExternalGrouping(&res, connsEmpty, src, dst,
-				&groupedCommonProperties{connDiff: connDiff, connStrKey: connDiffString})
-			if err != nil {
-				return err
+			if !(connDiff.conn1.IsEmpty() && connDiff.conn2.IsEmpty()) {
+				err := g.addLineToExternalGrouping(&res, src, dst,
+					&groupedCommonProperties{connDiff: connDiff, groupingStrKey: connDiffString})
+				if err != nil {
+					return err
+				}
 			}
 		}
 	}
 	g.appendGrouped(res)
 	return nil
 }
 
-func (g *GroupConnLines) addLineToExternalGrouping(res *[]*groupedConnLine, emptyConn bool,
-	src, dst VPCResourceIntf, commonProps *groupedCommonProperties) error {
-	if emptyConn {
-		return nil
+// group public internet ranges for explainability lines
+func (g *GroupConnLines) groupExternalAddressesForExplainability() error {
+	var res []*groupedConnLine
+	for _, rulesSrcDst := range *g.explain {
+		connStr := ""
+		if !rulesSrcDst.conn.IsEmpty() {
+			connStr = rulesSrcDst.conn.String() + semicolon
+		}
```

Comment:
```suggestion
		connStr := rulesSrcDst.conn.String() + semicolon
```
can also use `conn.String()` for an empty connection-set (string is "no connections")